compliance/collection/nodeinventorizer/nodeinventory.go

```diff
@@ -108,6 +108,26 @@ func convertAndDedupRHELComponents(rc *database.RHELv2Components) []*scannerV1.R
 	return maps.Values(convertedComponents)
 }
 
-func makeComponentKey(component *scannerV1.RHELComponent) string {
+func convertExecutables(exe []*scannerV1.Executable) []*storage.NodeInventory_Components_RHELComponent_Executable {
+	arr := make([]*storage.NodeInventory_Components_RHELComponent_Executable, len(exe))
+	for i, executable := range exe {
+		arr[i] = &storage.NodeInventory_Components_RHELComponent_Executable{
+			Path:             executable.Path,
```

Comment:
`executable.GetPath()`

Reply:
Good point, added in [fdbd8cd](https://github.com/stackrox/stackrox/pull/4392/commits/fdbd8cd232b3fa7ac139ce100ad7d65fb06e7793)